game/src/devtools/fifteen_min/mod.rs

```diff
@@ -74,7 +80,22 @@ impl State<App> for Viewer {
             Outcome::Clicked(x) => match x.as_ref() {
                 "close" => {
                     return Transition::Pop;
-                }
+                },
+                "About" => {
+                    return Transition::Push(PopupMsg::new(
+                        ctx,
+                        "About this OSM viewer",
```

Comment:
Figuring out what to say is the hard part of this task. :) For the title of this panel, "About 15-minute neighborhoods". For the contents.. What if you could access most of your daily needs with a 15-minute walk or bike ride from your house? Wouldn't it be nice to not rely on a climate unfriendly motor vehicle and get stuck in traffic for these simple errands? Different cities around the world are talking about what design and policy changes could lead to 15 minute neighborhoods. This tool lets you see what commercial amenities are near you right now, using data from OpenStreetMap.

And once we have some of the cooler features (overlay zoning codes, changing the codes and "upzoning" buildings), we can mention the bigger vision